src/parser.ts

```diff
@@ -88,42 +91,70 @@ export class Parser {
     }
 
     parseLaTeX(log: string) {
-        log = log.replace(/(.{78}(\w|\s|\d|\\|\/))(\r\n|\n)/g, '$1')
         this.buildLogRaw = log
-        const lines = log.replace(/(\r\n)|\r/g, '\n').split('\n')
+        const lines = log.split('\n')
         this.buildLog = []
+
+        let remainingEmptyLines = 0
+        const emptyResult = { type: '', file: '', text: '', line: undefined }
+        let currentResult: { type: string, file: string, text: string, line: number | undefined } = emptyResult
```

Comment:
You can remove `emptyResult` definition, and change `currentResult` like this:
```javascript
let currentResult: { type: string, file: string, text: string, line: number | undefined } = { type: '', file: '', text: '', line: undefined }
```
Then in subsequent `currentResult !== emptyResult` checks, change them to `currentResult.type !== ''`.

This is my observation, you can freely select adopting it or not.